javascript-timers/interval.js

```diff
@@ -0,0 +1,17 @@
+const h1 = document.querySelector('h1');
+
+setInterval(handleSetInterval, 1000);
+function handleSetInterval() {
+if (h1.textContent === '4') {
+  h1.textContent = '3';
+} else if (h1.textContent === '3') {
+  console.log('hi');
+  h1.textContent = '2';
+} else if (h1.textContent === '2') {
+  h1.textContent = '1';
+} else if (h1.textContent === '1') {
+  h1.textContent = '~Earth Beeeelooowww Us~';
+} else if (h1.textContent === '~Earth Beeeelooowww Us~') {
+  h1.textContent = '4';
+}
```

Comment:
The interval should stop after reaching the last phrase:

---

![image](https://user-images.githubusercontent.com/16143771/123709692-54e95e80-d822-11eb-92a5-05757b61d451.png)